### PR TITLE
feat: address commandhandling consumer

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: paket.dependencies [hunk 1]
```diff
@@ -43,8 +43,9 @@ nuget Be.Vlaanderen.Basisregisters.CommandHandling.SqlStreamStore 7.0.0
 nuget Be.Vlaanderen.Basisregisters.EventHandling 4.2.0
 nuget Be.Vlaanderen.Basisregisters.EventHandling.Autofac 4.2.0
 
-nuget Be.Vlaanderen.Basisregisters.MessageHandling.AwsSqs.Simple 4.3.2
-nuget Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple 4.3.2
+nuget Be.Vlaanderen.Basisregisters.MessageHandling.AwsSqs.Simple 4.6.1
+nuget Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Producer 4.6.1
```
Comment (jvandaal) [inline paket.dependencies:47]: Why do we need the producer here? We don't produce anything in building (yet)?

File: paket.dependencies [hunk 2]
```diff
@@ -43,8 +43,10 @@ nuget Be.Vlaanderen.Basisregisters.CommandHandling.SqlStreamStore 7.0.0
 nuget Be.Vlaanderen.Basisregisters.EventHandling 4.2.0
 nuget Be.Vlaanderen.Basisregisters.EventHandling.Autofac 4.2.0
 
-nuget Be.Vlaanderen.Basisregisters.MessageHandling.AwsSqs.Simple 4.3.2
-nuget Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple 4.3.2
+nuget Be.Vlaanderen.Basisregisters.MessageHandling.AwsSqs.Simple 4.7.0
+nuget Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Producer 4.7.0
+nuget Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple 4.7.0
```
Comment (ArneD) [inline paket.dependencies:48]: Simple still needed?

File: src/BuildingRegistry.Consumer.Address/BuildingRegistry.Consumer.Address.csproj [hunk 3]
```diff
@@ -18,12 +18,17 @@
...
   </ItemGroup>
 
   <ItemGroup>
     <Compile Include="..\..\SolutionInfo.cs" Link="Properties\SolutionInfo.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Migrations\" />
+  </ItemGroup>
+
```
Comment (ArneD) [inline src/BuildingRegistry.Consumer.Address/BuildingRegistry.Consumer.Address.csproj:32]: Remove

**Review by ArneD — CHANGES_REQUESTED** [review]

File: src/BuildingRegistry.Consumer.Address/ConsumerAddress.cs [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        private readonly ILoggerFactory _loggerFactory;
+        private readonly IIdempotentConsumer<ConsumerAddressContext> _kafkaIdemIdompotencyConsumer;
+        private readonly ILogger<ConsumerAddress> _logger;
+
+        public ConsumerAddress(
+            ILifetimeScope lifetimeScope,
+            IHostApplicationLifetime hostApplicationLifetime,
+            IDbContextFactory<BackOfficeContext> backOfficeContextFactory,
+            ILoggerFactory loggerFactory,
+            IIdempotentConsumer<ConsumerAddressContext> kafkaIdemIdompotencyConsumer)
```
Comment (jvandaal) [inline src/BuildingRegistry.Consumer.Address/ConsumerAddress.cs:29]: What does IdOmpotency mean? :-P

File: paket.dependencies (hunk 2 above)
Comment (jvandaal) [reply paket.dependencies:48]: Removed and stills builds.

File: src/BuildingRegistry.Consumer.Address/BuildingRegistry.Consumer.Address.csproj (hunk 3 above)
Comment (jvandaal) [reply src/BuildingRegistry.Consumer.Address/BuildingRegistry.Consumer.Address.csproj:32]: Removed.

**Review by ArneD — APPROVED** [review]

**Pull request merged**